src/controllers/sessionLockController.ts

```diff
@@ -414,6 +417,35 @@ export default class SessionLockController {
       ),
     ] as const;
 
+  private readonly clearInstallation = (fiscalCode: FiscalCode) =>
+    // async fire & forget
+    pipe(
+      TE.tryCatch(
+        () =>
+          this.notificationServiceFactory(fiscalCode).deleteInstallation(
+            fiscalCode
+          ),
+        (err) =>
+          Error(
+            `Cannot delete Notification Installation: ${JSON.stringify(err)}`
+          )
+      ),
+      TE.chainW(
```

Comment:
widen not needed. here we are returning `Error` in the left side as the step before
```suggestion
      TE.chain(
```